app/models/Broadcast.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const mongoose = require('mongoose');
+
+/**
+ * Schema.
+ */
+const broadcastSchema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  type: { type: String, enum: ['signup_prompt'] },
+  campaign: Number,
```

Comment:
Let's leave this as number until we're consuming content from a new API and ready to make the change globally -- it's defined as a `Number` in all of the other models.